protos/core.proto

```diff
@@ -186,13 +186,14 @@ message QueryEvaluationError {
   ErrorLocation error_location = 2;
 }
 
-message QueryEvaluationResponse {
+message QueryEvaluation {
   enum QueryEvaluationStatus {
     SUCCESS = 0;
     FAILURE = 1;
   }
   QueryEvaluationStatus status = 1;
   QueryEvaluationError error = 2;
+  string query = 3;
```

Comment:
i'm not super clear on why this is useful? i would expect callers to `evaluate` single queries/tables/assertions/operations individually.

Reply:
This will be needed to make line numbers match in the closed source UI I believe. As the statement that is run against the warehouse will only be fully built in the backend (when the `evaluate` part of `@dataform/api` runs `publishTasks`), this will need to be send back.